account_banking_pain_base/models/account_payment_order.py

```diff
@@ -657,7 +654,7 @@ def generate_remittance_info_block(self, parent_node, line, gen_args):
                 creditor_ref_info_type_issuer = etree.SubElement(
                     creditor_ref_info_type, "Issr"
                 )
-                creditor_ref_info_type_issuer.text = communication_type
+                creditor_ref_info_type_issuer.text = "ISO"
```

Comment:
@alexis-via please keep communication_type. We need it for belgium where (IIRC) it must become bba.
https://github.com/OCA/l10n-belgium/tree/10.0/l10n_be_iso20022_pain

cc/ @luc-demeyer

Reply:
Thanks for pointing this out. Let's talk about this during the next code sprint, because the datamodel for structured communication type is really messy and unclean. I'm confident that we'll agree on a nice and clear datamodel on this, that will make the l10n_be_iso20022_pain simpler and make the XML generation code simpler.